CITi-UFPE/hackapride_2022
# Feature(font-and-outsideContainer): Add fonts and outside container base

# What I did?

- Created base fonts and the outside container 

## How to test?

1. Open the terminal and go to project folder

1. In terminal:
   1. `git checkout develop`
   1. `git pull`
   1. `git checkout  feature/font-and-outsideContainer`
   1. `git pull origin feature/font-and-outsideContainer`
   1. `yarn install` ou `npm install`
   1. `code .`
   1. `yarn start` ou `npm start`

1. In browser go to http://localhost:3000/

1. Verify if the main page is with the same font as the mockup

1. Read section change files on GitHub

1. Test the things if you think that is necessary test, even those that are not in this description
